docs/en/reference/advanced-configuration.rst

```diff
@@ -71,9 +76,26 @@ Configuration Options
 The following sections describe all the configuration options
 available on a ``Doctrine\ORM\Configuration`` instance.
 
+Native Lazy Objects (***OPTIONAL***)
```

Comment:
The triple stars are bizarre and [render bizarrely](https://www.doctrine-project.org/projects/doctrine-orm/en/3.3/reference/advanced-configuration.html#proxy-directory-required), I'm going to file a PR against 2.x to fix that.